runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -227,28 +227,44 @@ getInternalTypeFromSignature(J9JavaVM *vm, J9Class *typeClass, U_8 sigType)
  *
  * @param argType the predefined signature type
  * @param argValue the requested argument value
- * @return a U_32 value for the normalized argument
+ * @return a I_32 value for the normalized argument
  *
  * Note:
- * The argument normalization is only intended for the type less then 4 bytes
+ * The argument normalization is only intended for types less than 4 bytes
  * in java, which include boolean (1 byte), byte, char (2 bytes), short.
```

Comment:
This comment is still misleading as it suggests the function is only used for the listed Java types when that is not the case.

Reply:
I think the comment already explains what need to be normalized except `int/float`. So  I don't think there is anything confusing in there.

Reply:
Updated the comment based on the offline discussion.